### PR TITLE
ShowFieldGlyphs Documentation Rewrite

### PR DESCRIPTION
**Pull request by tpat95**
Color mapping for tensors is based off magnitude of eigenvalues, not just the primary eigenvalue.

Edit(5/13):
-Rewrote page so it is more descriptive and has a cleaner layout.
-Added information on secondary and tertiary ports to match the changes in the module.

Reviewers:
@jessdtate 
@dcwhite

### CODE REVIEW

File: _includes/modules/ShowFieldGlyphs.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ This module visualizes the data that makes up a Field. When possible and selecte
...
+The field in the first input port holds the data that is to be visualized. By default is will be displayed using the the default color, which is editable from the GUI If there is a color map attached to the second input port, then the data can be used as an index into the color map, and the glyph is rendered with appropriate colors. In addition, the data itself can be converted into a color. **Scalar** data creates a gray scale mapping, **vector** data (normalized) creates RGB colors, and the **principle eigenvector**c (normalized) of tensor data also creates RGB colors. For tensors, the vector magnitude of the 3 eigenvalues is used as the index into the color map. The absolute value of vector x, y, z components are used to generate RGB color components:
```
Comment (dcwhite) [inline _includes/modules/ShowFieldGlyphs.md]:
```suggestion
The field in the first input port holds the data that is to be visualized. By default is will be displayed using the the default color, which is editable from the GUI If there is a color map attached to the second input port, then the data can be used as an index into the color map, and the glyph is rendered with appropriate colors. In addition, the data itself can be converted into a color. **Scalar** data creates a gray scale mapping, **vector** data (normalized) creates RGB colors, and the **principle eigenvector** (normalized) of tensor data also creates RGB colors. For tensors, the vector magnitude of the three eigenvalues is used as the index into the color map. The absolute value of vector x, y, z components are used to generate RGB color components:
```

Comment (dcwhite) [reply _includes/modules/ShowFieldGlyphs.md]: Fix typo

**Review by dcwhite — CHANGES_REQUESTED** [review]

**Review by jessdtate — COMMENTED** [review]
Does this description match what the colormap lookup currently does, or the way that it will be?

**Comment by dcwhite** [comment]
Not sure how to fix the build error. I'll rerun it...

**Comment by tpat95** [comment]
Dan, could you rerun the Travis build?

**Review by dcwhite — APPROVED** [review]

**Pull request merged**